src/rainmaker/view/item.ts

```diff
@@ -48,9 +48,18 @@ export class RMakerItem extends TreeItem {
   public set commandId(v: string) {
     this.command = {
       command: v,
-      title: typeof this.label === "string" ? this.label : this.label.label,
+      title:
+        typeof this.label === "string"
+          ? this.label
+          : this.label.label
+          ? this.label.label
+          : undefined,
       tooltip:
-        typeof this.tooltip === "string" ? this.tooltip : this.tooltip.value,
+        typeof this.tooltip === "string"
+          ? this.tooltip
+          : this.tooltip.value
```

Comment:
this will still cause error... because 

```js
if (typeof this.tooltip === "string") {
    tooltip = this.tooltip;
} else if ( this.tooltip.value) {
               //^--- crash here since this.tooltip is undefined
    tooltip = this.tooltip.value;
} else { 
    tooltip = undefined;
}
```